ui/bootstrap/model.go

```diff
@@ -17,14 +17,14 @@ const (
 
 const InstallQuestionMsg = `# Installing A Node
 
-It looks like you're running this for the first time. Would you like to install a node? (Y/n)
+It looks like you're running this for the first time. Would you like to install a node? (Y/N)
```

Comment:
```suggestion
It looks like you're running this for the first time. Would you like to install a node? (y/n)
```
no default here - y/n